### DISCUSSION TITLE
Can you provide an example configuration for a Sleep as android device

### DISCUSSION THREAD

**Discussion by atavako5** [IATkachenko/HA-SleepAsAndroid#24] [Q&A] [answered]
Hello Igor and contributors,

I'm trying to set up your integration but I'm kind of stuff at the blueprint where it asking for a SleepAsAndroid device. It keeps saying No matching devices found, please let me know what I need to add to my config.yaml or anywhere else to create a device. I've looked through the code and the documentation and I can't find anything. Keep in mind that I'm very new to HA and just trying to figure things out :)

Thank You,

Ali

**Comment by IATkachenko** [comment, marked as answer]
@atavako5, hello.

You don't need any special in `configuration.yaml`.
Just add the integration to you setup via user interface (instruction is avaliable in README) and make sure that integration may catch MQTT events from you application. Device should appear after first event from application.

Then you will be able to select the device in bluprint.
